src/SIS_dyn_cgrid.F90

```diff
@@ -1024,13 +1059,22 @@ subroutine SIS_C_dynamics(ci, mis, mice, ui, vi, uo, vo, &
         drag_u = cdRho * sqrt(uio_init**2 + v2_at_u )
       endif
       if (drag_max>0.) drag_u = min( drag_u, drag_max )
+      if (CS%lemieux_landfast) drag_bot_u = CS%Tb_u(I,j) / &
```

Comment:
If drag_bot_u were simply combined with drag_u, there would not be a need for a duplicated block at line 1067.